specification/paths/Returns-v1-return-methods.json

```diff
@@ -17,14 +17,13 @@
         "$ref": "#/components/parameters/query-filter-shop"
       },
       {
-        "allOf": [
-          {
-            "$ref": "#/components/parameters/query-filter-weight"
-          },
-          {
-            "deprecated": true
-          }
-        ]
+        "name": "filter[weight]",
+        "in": "query",
+        "deprecated": true,
+        "description": "Weight value in grams to filter by.",
+        "schema": {
+          "type": "number"
+        }
```

Comment:
I had to define it like that because merging with deprecated flag was making the entire filter disappear from the rendered spec.